### ISSUE TITLE
Brand New Screen Reader: Looking for the interested developers for FOSS project.

### ISSUE THREAD

**Issue by akash07k** [talkback-foss-team/talkback-foss#28]
Hey all,
So I thought that it is the best repo to post this hence I'm here with this.
Please read it carefully:
Actually Talkback has several flaws and the code is kind of quite legacy.
Jieshuo is comprehensive, but is closed source.
I'm planning to develop a very comprehensive, stable, powerful and feature rich screen reader for android which should be open source and can replace both the existing leading screen readers.
As there's NVDA on Windows, I want to implement the same for android too.
So, I'm looking for the interested developers.
Please let me know if you are interested.
Android Studio is quite slow and crappy from accessibility point of view, so my main target is to develop such screen reader purely in Xamarin .net.
But if I get many developers who are interested in Kotlin/Java(Android Studio) then I will consider that too.
However, .NET developers are more preferred for this project.
Please get in touch with me if you or somebody is interested and please share this post on your groups/mailing lists too so that we can get enough interested developers and contributers.
If you are really interested, then please fill this short and quick form:
https://forms.gle/fU3gEZDDWSAHFhfr7

**Comment by PatrykMis** [comment]
Great iniciative which I will support.

Better something than nothing, but unfortunately I think Xamarin Will not be as flexible as Kotlin, because it will add additional translation layer (speed/performance issues), may require to use non-free software and/or libraries and we would unable to use native Android APIs.

**Comment by PatrykMis** [comment]
Regarding Android Studio accessibili - it performs quite well with screen readers (I'm using JAWS and NVDA). Also studying some documentation might help:

* [Accessibility | IntelliJ IDEA Documentation](https://www.jetbrains.com/help/idea/2022.2/accessibility.html) (as it's based on IntelliJ Idea)
* [Accessibility features  |  Android Studio ](https://www.jetbrains.com/help/idea/2022.2/accessibility.html)

**Comment by PatrykMis** [comment]
Also, take a look at @Irineu333's [SpeekTouch](https://github.com/Irineu333/SpeakTouch) project as it's really promising.

Closing.

**Comment by akash07k** [comment]
🤣 Do you think that the person who has spent so many years with Android Studio hasn't checked these documentations before?
> Regarding Android Studio accessibility - it performs quite well with screen readers (I'm using JAWS and NVDA). Also studying some documentation might help:
> 
> * [Accessibility | IntelliJ IDEA Documentation](https://www.jetbrains.com/help/idea/2022.2/accessibility.html) (as it's based on IntelliJ Idea)
> * [Accessibility features  |  Android Studio ](https://www.jetbrains.com/help/idea/2022.2/accessibility.html)

**Comment by akash07k** [comment]
Yes, you are absolutely right.

Braille displays are a godsend especially for coding because they give 
us a perfect view of indentations, special characters etc.

I don't know this guy, I'll read his post.

If you are interested and comfortable, let's connect on 
WhatsApp/telegram so that we can both can communicate in depth and 
exchange the ideas in realtime.

Let me know how can I share my TG username with you.

**Comment by PatrykMis** [comment]
Sure. Drop me an e-mail - it's on my public GitHub profile.
